updater/Gemfile.lock

```diff
@@ -1,19 +1,19 @@
 PATH
   remote: ../bundler
   specs:
-    dependabot-bundler (0.211.0)
-      dependabot-common (= 0.211.0)
+    dependabot-bundler (0.212.0)
+      dependabot-common (= 0.212.0)
```

Comment:
This is a bit scary... `Gemfile.lock` was way back on `0.211.0`, yet we released `0.212.0` on Sept 5th: https://github.com/dependabot/dependabot-core/releases/tag/v0.212.0

Ideally we're actually using the _local_ copy of `dependabot-common`, so I'm unclear why this is even pinned here? Or at least I hope we're using the _local_ copy available as part of the build and not fetching it off `RubyGems`?? 😱 

But if it does need to be pinned for some reason, then we ought to at least have it keep up with the latest released version... Maybe that's a secondary step in our release script after we tag a new version?

Reply:
These are pulled in as path dependencies, not from rubygems. The version shows whatever the version is in the branch you're working in, but so does the rest of the code, it always has all the latest changes from disk

Reply:
I guess we could have a CI step that runs `bundle update --bundler` and reminds us to commit the result whenever this lock file is about to get out of date (or directly synchronizes it).

Reply:
IIUC, https://github.com/dependabot/dependabot-core/pull/5972 solved this by adding `bundle config set --local frozen 'true'` so that CI will blow up if this lock file gets out of date...

Is my understanding correct @deivid-rodriguez ? Just want to double-check before I mark this conversation as resolved... if not, I'll look into adding a CI step in a follow-on PR.

Reply:
Gotcha, then for now we're probably good enough with the comment I added in https://github.com/dependabot/dependabot-core/pull/5858/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R77.

It may get overlooked by someone bumping bundler, but every so often at least someone will see that and remember to do it hopefully so we shouldn't get too far behind.

It'd certainly be nice to have a CI enforcement step, but I'm not coming up with a trivial way to do that that isn't a bit brittle (regex comparisons of the two versions?), and the delta generally won't cause breakage so IMO the comment can suffice for now.